packages/format-currency/src/index.ts

```diff
@@ -65,17 +67,26 @@ function getCachedFormatter( {
 		}
 	}
 
-	const formatter = new Intl.NumberFormat( addNumberingSystemToLocale( locale ), {
-		style: 'currency',
-		currency,
-		// There's an option called `trailingZeroDisplay` but it does not yet work
-		// in FF so we have to strip zeros manually.
-		...( noDecimals ? { maximumFractionDigits: 0, minimumFractionDigits: 0 } : {} ),
-	} );
+	try {
+		const formatter = new Intl.NumberFormat( addNumberingSystemToLocale( locale ), {
+			style: 'currency',
+			currency,
+			// There's an option called `trailingZeroDisplay` but it does not yet work
+			// in FF so we have to strip zeros manually.
+			...( noDecimals ? { maximumFractionDigits: 0, minimumFractionDigits: 0 } : {} ),
```

Comment:
I wonder if it makes sense for the `Intl.NumberFormat` settings to be customizable when calling `formatCurrency()`.

Reply:
By allowing passing in arbitrary options to `NumberFormat`? 🤔 I'm not sure. I think that makes for a bit of a leaky abstraction. If you want to use `NumberFormat`, a caller can just use it directly without needing this function. I _do_ think that there's the potential for better abstractions in this library in the future, like functions that perform some of these options by default but for now I'd opt to keep the options already present in `formatCurrency` unchanged in case we ever want to do things differently.